Autopilot/TelemetryManager/Telemetry_Manager.c

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file Telemetry_Manager.c
+ * @author Jingting Liu
+ * @date January 14th, 2021
+ */
+
+#include <stdio.h>
+#include <common/mavlink.h>
+
+#include "telemetry_state_machine.h"
+
+
+
+TM_STATE_e current_state = INIT;
```

Comment:
variables like this should be declared static, so that they're not global. But this doesen't matter, cuz you should be doing this in a cpp way anyway.